src/main/java/io/github/arrayv/frames/UtilFrame.java

```diff
@@ -293,8 +310,9 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
                                     .addComponent(this.jButton3, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
                                     .addComponent(this.jButton2, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
                                     .addComponent(this.jButton1, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                                    .addComponent(runScriptButton, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
```

Comment:
feel the gap between the run script button and the sorting model combo box is too small; also feel maybe it fits better where sorting stuff is, but I guess the scripting language is meant to have a broader scope

Reply:
What should the gap be instead? I really had no idea how to put the Run Script button in, and I just kinda put it wherever.

Reply:
But it is the same size as other gaps.
![image](https://user-images.githubusercontent.com/42721887/158026581-53816ef1-f14c-44fe-bcac-5be5cda20c09.png)

Reply:
it's actually slightly smaller than those

and personally I don't think it fits in the same "group" as the combobox

Reply:
But in what "group" does it belong? I put it there because I felt like it was an overarching thing for the entire program.

Reply:
perhaps at the very bottom? the combobox and below visual settings fit together, but putting it above the combobox feels awkward

Reply:
The bottom makes sense, I guess. That was the original plan, actually.

Reply:
Is this better?

Reply:
looks nice, though I feel it's a bit close to the bottom edge
![image](https://user-images.githubusercontent.com/69981203/158033449-fb6992b2-677e-4aa3-9fa1-8fb2aec8ffaa.png)

Reply:
Does this look better?